lib/domains/character/types.tsx

```diff
@@ -182,14 +184,37 @@ export type IImageBlock = {
   value: string;
 };
 
+export type ILinkBlock = {
+  type: BlockType.Link;
+  meta: {
+    helperText?: string;
+    displayName?: string;
```

Comment:
**question** I'm curious why you decided to add a new meta property for the link display name instead of using the `block.label` property ?

I understand that the label isnt usually used for this behavior but I was wondering if you considered it and what made you pick this option

Reply:
Good point! I think perhaps I went this direction before I had fully realized I could use the block.label property. I'll swap that out : )